test/unit/hpa.c

```diff
@@ -1416,10 +1432,12 @@ TEST_BEGIN(test_hpa_hugify_style_none_huge_no_syscall) {
 	nstime_init(&defer_curtime, 10 * 1000 * 1000);
 
 	tsdn_t *tsdn = tsd_tsdn(tsd_fetch());
-	/* First allocation makes the page huge */
+	/* First allocation makes the page huge iff THP are in 'always' mode */
```

Comment:
The comment here is a bit confusing. I guess what we want to say here is "Allocating more than 1/4 of a hugepage to enable hugification on a 2MB unit."?